src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,19 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine truck = new Truck();
+        Machine bulldozer = new Bulldozer();
+        Machine excavator = new Excavator();
 
+        Machine[] machines = new Machine[3];
+        machines[0] = truck;
+        machines[1] = bulldozer;
+        machines[2] = excavator;
+
+        for (Machine machine : machines) {
+            machine.doWork();
+            machine.stopWork();
+        }
```

Comment:
It's important to have informative messages in methods. Ensure that the messages indicate what type of Machine is working, such as 'Truck', 'Bulldozer', or 'Excavator'. This requires the `doWork` and `stopWork` methods in the respective classes to output specific messages about the machine type.